README.md

```diff
@@ -204,3 +204,59 @@ The following parameters refer to components that are not internal to Codacy, bu
 You can also configure values for the PostgreSQL database via the Postgresql [README.md](https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
 
 For overriding variables see: [Customizing the chart](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing)
+
+## Development
+
+### Set up your environment for DigitalOcean clusters
```

Comment:
Remove one of the "#".

Reply:
This is supposed to be a subsection of the "Development" section. 

I think, I can just write an intro to this section maybe. Would that be ok?

Reply:
Instead of having a section called "Set up your environment for DigitalOcean clusters" at the same level of the numbered steps, right? That would work.

Reply:
Then maybe the numbered sections below this one should be one level deeper, then?